compiler/ast.nim

```diff
@@ -853,6 +853,8 @@ type
   TSym* {.acyclic.} = object of TIdObj # Keep in sync with PackedSym
     # proc and type instantiations are cached in the generic symbol
     case kind*: TSymKind
+    of skModule:
+      realModule*: PSym # for `createModuleAlias`
```

Comment:
I know you want me to review and accept it, but I'm still thinking about the implications of this data model extension.

Reply:
This is not required and harmful for IC.

Reply:
=> https://github.com/nim-lang/Nim/pull/18362